packages/astro/src/integrations/index.ts

```diff
@@ -221,6 +221,17 @@ export async function runHookConfigDone({
 									);
 								}
 							}
+							if (!validationResult.assets) {
+								info(
+									logging,
+									'astro',
+									`The selected adapter ${adapter.name} does not support Sharp or Squoosh as image services. To ensure your project still build, image processing has been disabled.`
```

Comment:
```suggestion
									`The selected adapter ${adapter.name} does not support Sharp or Squoosh for image optimization. To ensure your project is still able to build, image optimization has been disabled.`
```

Reply:
I on purpose avoid the word "optimization" because this also affects resizing your images for instance.